delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
```

Comment:
Implement prepared statements for executing the delete query and consider adding a confirmation step before deletion to avoid accidental removals.